### PR TITLE
ref(ts): Convert PlatformIconTile to typescript

### PR DESCRIPTION
**Pull request by matejminar**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/platformIcon.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const PlatformIcon = ({platform, size, ...props}: Props) => {
     // TODO(Priscila): find a better way to do it, maybe by removing the react svg path fill attributes
     return (
       <StyledPlatformIconTile
-        platform={platform}
+        platform={platform as any} // TODO(ts): this will be removed once new platformicons land
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/platformIcon.tsx:85]: 👍

File: src/sentry/static/sentry/app/components/platformIconTile.tsx [hunk 2]
```diff
@@ -78,9 +80,15 @@ const getColorStyles = ({monoTone, platform}) => {
   );
 };
 
+type Props = {
+  platform: keyof typeof PLATFORM_ICONS;
+  className?: string;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/platformIconTile.tsx:85]: I would move it to the last position of this object

**Review by priscilawebdev — APPROVED** [review]

File: src/sentry/static/sentry/app/components/platformIconTile.tsx (hunk 2 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/platformIconTile.tsx:85]: It looks like it is in the right spot to me. monoTone is also an optional prop.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/platformIconTile.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/platformIconTile.tsx:85]: lol..I think I missed that

**Pull request merged**